dap/BESDapResponseBuilder.cc

```diff
@@ -1094,42 +1205,31 @@ void BESDapResponseBuilder::send_dap4_data_using_ce(ostream &out, DMR &dmr, bool
 
 void BESDapResponseBuilder::send_dap4_data(ostream &out, DMR &dmr, bool with_mime_headers)
 {
-    try {
```

Comment:
Is this part of the alarm branch? Or is it something else that probably should be in another branch?